wp-content/themes/init_theme_name/admin/class-admin.php

```diff
@@ -94,7 +94,7 @@ public function enqueue_scripts() {
    *
    * @since 2.0.0
    */
-  function set_enviroment_body_class( $classes ) {
+  public function set_enviroment_body_class( $classes ) {
```

Comment:
@iruzevic Should we replace this with:

```
  /**
   * Method that changes admin colors based on environment variable
   *
   * @param string $color_scheme Color scheme string.
   * @return string              Modified color scheme.
   *
   * @since 1.1.0
   */
  public function set_admin_color_based_on_env( $color_scheme ) {
    if ( ! defined( 'INF_ENV' ) ) {
      return false;
    }
    if ( INF_ENV === 'production' ) {
      $color_scheme = 'sunrise';
    } elseif ( INF_ENV === 'staging' ) {
      $color_scheme = 'blue';
    } else {
      $color_scheme = 'fresh';
    }
    return $color_scheme;
  }
```

This changes the admin based on the environment.